jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/es/ESManager.java

```diff
@@ -760,6 +761,31 @@ public static void uploadData(String index, List<Map<String, Object>> docs) {
         }
     }
 
+    public static void uploadVulnerabilityData(String index, List<Map<String, Object>> docs) {
+        String actionTemplate = "{ \"index\" : { \"_index\" : \"%s\" , \"_id\" : \"%s\" } }";
```

Comment:
Are we creating the child docType here? If yes, we need to add:
1. Routing, usually the parent id.
2. Relations object for child documents.

Reply:
@santhosh-challa we are not having a docType field. We are uploading all the collected vulnerabilities data into the index crowdstrike_vulnerability. At that point we dont know the relation between the parent and child docs. Once the managed policy runs, vulnerabilities for each instanceId are queried and violation is created

Reply:
Ok. Qq, if you know the particular asset id cant we create a docType in same index like Qualys?

Reply:
@santhosh-challa , Paul suggests this way is a more streamlined approach to store the vulnerabilities in a separate index, rather than storing them in the existing index(aws_ec2) with a new docType

Reply:
@ershad-paladin default setting for the number of shards is 5 in OpenSearch. Please use 1 shard for now as 5 will be too much unless the data is huge. For now, we are using 1 shard as default for all of our indices. 
cc @plyubich

Reply:
@santhosh-challa I 'll make the change in my upcoming PR